ArcadiaParties-SPA/src/app/department/department-selector/department-selector.component.scss

```diff
@@ -0,0 +1,5 @@
+:host {
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
```

Comment:
I suppose it is better to have style defining how the component is placed near other components in their container. Component shouldn't know where it is used and how it should behave. Component is responsible only for its own internal styles.

Reply:
I've changed selector positioning